src/screens/CreatePost/index.js

```diff
@@ -733,7 +733,6 @@ const CreatePost = () => {
                 isNotFromHomeTab: true
               });
             }}
-            isToggleDisabled={params.followType === 'incognito'}
           />
           <Gap style={styles.height(8)} />
           <CreatePostInput
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Removing the `isToggleDisabled` prop from the component seems to be a good decision if it's no longer needed. However, ensure that this doesn't affect other parts of your application where this prop might be used or expected. If there are no such dependencies, then this change is fine.
```diff
-            isToggleDisabled={params.followType === 'incognito'}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->